pages/dashboard/index.js

```diff
@@ -52,7 +53,19 @@ const Dashboard = ({ session, friendData, spaceCardData }) => {
   const { peakStudyTimes, studyTimes } = chartData;
   const [profileOpen, setProfileOpen] = useState(false);
   const [open, setOpen] = useState(false);
-  const setClient = useSetRecoilState(clientState.client);
+  const [client, setClient] = useRecoilState(clientState.client);
+  const gqlClient = useApolloClient();
+  const GET_USERS = gql`
+    query {
+      users(userIds: ["60ff51c8684e9e2206a83bfb", "609ccafca1c3fe54cca40121"]) {
+        name
+        email
+        image
+      }
+    }
+  `;
+  const { loading, error, data } = useQuery(GET_USERS);
+  console.debug('data', data);
```

Comment:
I wouldn't put a console.debug here. It's gonna execute every rerender. Either put it somewhere else in a callback function for useQuery (reference `onCompleted` https://www.apollographql.com/docs/react/data/queries/ or delete it all together.